src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -746,6 +750,9 @@ public function getSqlType($type, $limit = null)
             case static::PHINX_TYPE_JSON:
             case static::PHINX_TYPE_JSONB:
             case static::PHINX_TYPE_UUID:
+            case static::PG_TYPE_CIDR:
+            case static::PG_TYPE_INET:
+            case static::PG_TYPE_MACADDR:
```

Comment:
We really try to avoid these whenever possible. The JSON(B) types are default types, even though not all adapters support them. I'd rather see the same done here.